engines/mbic3.h

```diff
@@ -24,26 +24,19 @@
 
 namespace pono {
 
-// Both clauses and cubes can be represented as vectors of predicates
-// They are just negations of eachother
-
-struct Clause
+/** Represents a conjunction of boolean terms
+ *  Keeps the conjuncts in a vector as well
+ *  and sorts them
+ */
+struct Conjunction
 {
-  Clause() {}
-  Clause(const smt::SmtSolver & solver, const smt::TermVec & lits);
-  smt::TermVec lits_;  // list of literals sorted by hash
-  smt::Term term_;     // term representation of literals as disjunction
-};
-
-struct Cube
-{
-  Cube() {}
-  Cube(const smt::SmtSolver & solver, const smt::TermVec & lits);
+  Conjunction() {}
+  Conjunction(const smt::SmtSolver & solver, const smt::TermVec & lits);
   smt::TermVec lits_;  // list of literals sorted by hash
```

Comment:
this should be conjunts_

Reply:
also in the constructor

Reply:
Good catch! Fixed with eb808a090671e330a379b06844ee32f975a69e87